gulpfile.js

```diff
@@ -82,83 +82,102 @@ function octofarmImg() {
     .pipe(gulp.dest("views/images"));
 }
 function vendorJS() {
-  return src("client_src/js/vendor/*")
-    // .pipe(sourcemaps.init({ loadMaps: true }))
-    // .pipe(sourcemaps.write("./"))
-    .pipe(gulp.dest("views/assets/js/vendor"));
+  return (
+    src("client_src/js/vendor/*")
+      // .pipe(sourcemaps.init({ loadMaps: true }))
```

Comment:
Enable sourcemaps again all the way to the bottom. Otherwise JS is not gonna be nice to debug.